sql/core/src/main/scala/org/apache/spark/sql/execution/command/cache.scala

```diff
@@ -83,13 +83,10 @@ case class UncacheTableCommand(
 /**
  * Clear all cached data from the in-memory cache.
  */
-case class ClearCacheCommand() extends RunnableCommand {
+case object ClearCacheCommand extends RunnableCommand with IgnoreCachedData {
```

Comment:
Hi, @maropu .
The purpose and title doesn't match. What you want is `IgnoreCahedData` and the test case, isn't it?

Reply:
Please update JIRA/PR together.